python/ray/services.py

```diff
@@ -1177,21 +1177,21 @@ def start_raylet(redis_address,
 
     command = [
         RAYLET_EXECUTABLE,
-        raylet_name,
-        plasma_store_name,
-        str(object_manager_port),
-        str(node_manager_port),
-        node_ip_address,
-        gcs_ip_address,
-        gcs_port,
-        str(num_initial_workers),
-        str(maximum_startup_concurrency),
-        resource_argument,
-        config_str,
-        start_worker_command,
-        java_worker_command,
-        redis_password or "",
-        temp_dir,
+        "-raylet_socket_name={}".format(raylet_name),
+        "-store_socket_name={}".format(plasma_store_name),
+        "-object_manager_port={}".format(object_manager_port),
+        "-node_manager_port={}".format(node_manager_port),
+        "-node_ip_address={}".format(node_ip_address),
+        "-redis_address={}".format(gcs_ip_address),
+        "-redis_port={}".format(gcs_port),
+        "-num_initial_workers={}".format(num_initial_workers),
+        "-maximum_startup_concurrency={}".format(maximum_startup_concurrency),
+        "-static_resource_list={}".format(resource_argument),
+        "-config_list={}".format(config_str),
+        "-python_worker_command={}".format(start_worker_command),
+        "-java_worker_command={}".format(java_worker_command),
```

Comment:
worker command contains spaces, will this be an issue?

Reply:
note that we can pass `"-java_worker_command"` and `java_worker_command` as 2 separate args, then we don't need to worry about spaces.

Reply:
It works fine. The value of `java_worker_command` can be passed into raylet smoothly in this way.(The reason is similar to the 2 separated args you mentioned.)